honeybee_energy/aperturetype.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+"""Aperture Types."""
+from honeybee.aperturetype import _ApertureType
+from honeybee.typing import float_in_range
+
+
+class OperableWindow(_ApertureType):
+    __slots__ = ('_fraction_operable')
```

Comment:
You are definitely right about the usage of tuples and you are also right to point out the code that I have here is ambiguous and should be changed.  However, I would also point out that `__slots__` doesn't always need to be formatted as a tuple, particularly when there is only one variable that I am adding here. So it would all still work if I just had the single string (without any parentheses).  But I agree the tuple is clearer if we ever wanted to add more variables so I will implement that.